### DISCUSSION TITLE
Close pop window by `run-shell-command`

### DISCUSSION THREAD

**Discussion by wutchzone** [helix-editor/helix#4879] [Q&A] [answered]
Hello, sorry for opening the issue, but I find myself in trouble closing the pop window, which is opened by `run-shell-command`. When I execute, for example, `run-shell-command echo Hello there`, then the pop window gets opened and I cannot find a way how to close it unless I close the whole editor. Could you please suggest me any action that would help me to close the generated window? Thank you.

**Comment by usrme** [comment, marked as answer]
`Ctrl-c` also applies as described in the FAQ for [closing popups](https://github.com/helix-editor/helix/wiki/FAQ#close-the-lsp-documentation-popup).

**Reply by usrme** [reply]
I've found that `Esc` also works, for me at least.
